### ISSUE TITLE
Error in computing centerline in VMTK

### ISSUE THREAD

**Issue by Nasrollah** [vmtk/vmtk#134]
Hello
I wanted to use Meshing based on centerlines tool in order to create a good mesh but when I was working with vmtk tutorial and this comment "vmtkcenterlines -seedselector openprofiles -ifile aneu_int.stl -ofile centerlines.vtp" which is based on an example of Gmesh WIki , I mean this page " https://geuz.org/trac/gmsh/wiki/FSIMesh", I got this error:  
ERROR: In C:\prova64-vmtk\vmtk-build\VTK\IO\vtkSTLReader.cxx, line 446
vtkSTLReader (0000000004AA34D0): STLReader error reading file: aneu_int.stl Premature EOF while reading end solid.
I was wondering if you answer to this question as soon as possible.
Thank you 
Nasrollah

**Comment by lantiga** [comment]
Hi, apparently the stl file is corrupted (truncated?). The error should be reproducible with just
```
vmtksurfacereader -ifile aneu_int.stl
```
Can you open the file with any other software (e.g. gmesh or paraview) for a check?

**Comment by Nasrollah** [comment]
Hi, The file could be opened with paraview  or gmesh software very well. However, I wanted to reproduce the centerline file on my own which is available in this page " https://geuz.org/trac/gmsh/wiki/FSIMesh", I used this comment "vmtkcenterlines -seedselector openprofiles -ifile aneu_int.stl -ofile centerlines.vtp" but unfortunately, I got the previous error.

**Comment by lantiga** [comment]
Can you send the file over? I'll take a look.

Luca

**Comment by Nasrollah** [comment]
this file can be downloaded from this page " https://geuz.org/trac/gmsh/wiki/FSIMesh"

**Comment by lantiga** [comment]
Thanks

**Comment by lantiga** [comment]
I downloaded the file and saw the error. The error is the result of a bug in VTK that has been fixed in VTK 6, see [http://www.vtk.org/Bug/view.php?id=13160](this bug report). It doesn't show with most STL files.
Work for porting vmtk to VTK 6 is underway.

In any case, the pipe you reported produces the following correct centerlines.
![screen shot 2015-04-06 at 18 00 15](https://cloud.githubusercontent.com/assets/191033/7007099/cd134342-dc86-11e4-873e-e59706a11e54.png)
